### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Tshobohwa**
# FEATURES ADDED
-footer contact form
-desktop version

### CODE REVIEW

**Comment by Tshobohwa** [comment]
# PARTNER CHALLENGE.
Dear reviewer, please consider the fact that unfortunately, I did not have any available partner this week, and consequently, I completed alone this activity.

File: index.html [hunk 1]
```diff
@@ -1,177 +1,241 @@
...
+          <li class="header-li">
+            <a href="header-link">Portfolio</a>
+          </li>
+          <li class="header-li">
+            <a href="header-link">About</a>
+          </li>
+          <li class="header-li">
+            <a href="header-link">Contact</a>
+          </li>
+        </ul>
```
Comment (leonardodiasb) [inline index.html]: - [ ] Your navbar is good and very similar to the Figma template. But kindly reduce the font-size of the items to match the Figma template and look more professional.
 - [ ] Also please use the proper id of the sections in the `href` attribute of the nav items to redirect the user to the correct part of the page. Currently, this is making the user leave the page and this is not a good UX.

File: style.css [hunk 2]
```diff
@@ -241,3 +254,177 @@ ul {
...
+
+  .header-li {
+    margin: 0 1rem;
+  }
+
+  #section-1 {
+    background-image: url('images/header-background-desktop.png');
+    background-size: auto;
+    padding: 5rem 25rem;
+  }
```
Comment (leonardodiasb) [inline style.css]: - [ ] Kindly reduce the padding left and right to make your page looks more like the desktop version of the Figma template. There is a lot of whitespace on the sides of this section.
 - Your version:
 - 
image

 - Figma template:
 - 
image

File: index.html [hunk 3]
```diff
@@ -1,177 +1,241 @@
...
+              </h3>
+              <p class="work-summary">
+                  A daily selection of privately personalized reads; no accounts or
+                  sign-ups required.
+              </p>
+              <ul class="work-used-tools">
+                  <li class="tool">html</li>
+                  <li class="tool">css</li>
+                  <li class="tool">javascript</li>
+              </ul>
```
Comment (leonardodiasb) [inline index.html]: - [ ] Kindly reduce the font-size of the technologies list in all the cards to match the Figma template. On the template, the font-size is smaller than the rest of the content of the card.
 - Your design:
 - 
image

 - Figma template:
 - 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Tshobohwa, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Correct use of GitHub flow. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS 🔴 

Hi @Tshobohwa 👍 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

invalid

## Invalid Code Review Request

You have submitted a pull request with the wrong repository address. Kindly consider checking again from your end and submit the appropriate link. You submitted a link to your `Desktop verison` instead of the `Contact section`.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @Tshobohwa , 

I am Favour 👩🏽 , your reviewer for this PR.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**